src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/ArtifactoryGradleBuild/help-gradleBuild.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Build with Gradle, while resolving dependencies and deploying artifacts from and to Artifactory.
```

Comment:
This is a weird step parameter, and I think that someone who is involved with the antifactory plugin will best be able to give some extra details of what this parameter is.